vcd/resource_vcd_vapp.go

```diff
@@ -100,6 +101,11 @@ func resourceVcdVApp() *schema.Resource {
 				Optional: true,
 				Default:  true,
 			},
+			"properties": {
```

Comment:
We had this idea of keeping vApp thin... What do guest properties on a vApp (as opposed to VM) actually do? Thanks.

Reply:
Well. There is a concept of vApp template. In some cases it can be a bundle of VMs for an app. In we don't support much of "vApp deployments" so far so there is a question for its usability.

UI also has it:
<img width="528" alt="image" src="https://user-images.githubusercontent.com/15804230/63692481-6fafae00-c81a-11e9-9de0-5b108266da01.png">